gammapy/time/lombscargle_gammapy.py

```diff
@@ -0,0 +1,171 @@
+# from lombscargle
+from astropy.stats import LombScargle
+# from PyAstronomy.pyTiming.pyPeriod import Gls
+# from PyAstronomy.pyTiming import pyPeriod
```

Comment:
Please remove commented out code and all print statements.
Rename `lombscargle_gammapy.py` to `lombscargle.py`